ElasticApmAgent.sln

```diff
@@ -51,6 +51,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Elastic.Apm.AspNetFullFrame
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Elastic.Apm.Tests.MockApmServer", "test\Elastic.Apm.Tests.MockApmServer\Elastic.Apm.Tests.MockApmServer.csproj", "{AFFB238B-10C0-4229-9527-184EFCF0F5AC}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Elastic.Apm.Tests.Data", "test\Elastic.Apm.Tests.Data\Elastic.Apm.Tests.Data.csproj", "{BE2156EB-6C2C-4D13-925D-7794713C03ED}"

```

Comment:
You can share `TransactionMaxSpansTestData` between `Elastic.Apm.Tests` and `Elastic.Apm.AspNetCore.Tests` by placing `TransactionMaxSpansTestData` class in `Elastic.Apm.Tests` (for example in new `Data` sub-directory under `Elastic.Apm.Tests` directory and `Elastic.Apm.Tests.Data` namespace). It will work because `Elastic.Apm.AspNetCore.Tests.csproj` references `Elastic.Apm.Tests.csproj`. I don't think it's worth creating a new project just for this. WDYT?

Reply:
Agree with you. I didn't notice reference to Elastic.Apm.Tests.csproj by those projects. Will fix it.